guidelines/MONAI-Workloads.md

```diff
@@ -0,0 +1,1149 @@
+
+**Importance to Clinical AI Deployment:** medium, the use case will drive up the quality and model generalization but will not directly impact AI use as part of the radiology workflow.
+
+**Figure:** Retrospective Study
+
+![MONAI_Retrospective Study](assets/MONAI_Retrospective_Study.jpg)
+
+
+### Clinical Workloads Summary {#clinical-workloads-summary}
+
+
+<table>
+  <tr>
+   <td><strong>Workload</strong>
+   </td>
+   <td><p style="text-align: right">
+<strong>Payload</strong></p>
+
+   </td>
+   <td><p style="text-align: right">
+<strong>Pixels / Slice</strong></p>
+
+   </td>
+   <td><p style="text-align: right">
+<strong>bpp</strong></p>
+
+   </td>
+   <td><strong>Response Time</strong>
+   </td>
+   <td><p style="text-align: right">
+<strong>Data Size</strong></p>
+
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Emergency Diagnostics</strong>
+   </td>
+   <td><p style="text-align: right">
+1 study / 1  series</p>
+
+   </td>
+   <td><p style="text-align: right">
+512 x 512 x 100</p>
+
+   </td>
+   <td><p style="text-align: right">
+16</p>
+
+   </td>
+   <td>&lt; 10 minutes
+   </td>
+   <td><p style="text-align: right">
+~260MB</p>
+
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Scheduled Radiology</strong>
+   </td>
+   <td><p style="text-align: right">
+1 Study</p>
+
+   </td>
+   <td><p style="text-align: right">
+512 x 512x2000</p>
+
+   </td>
+   <td><p style="text-align: right">
+16</p>
+
+   </td>
+   <td>> 30 minutes
+   </td>
+   <td><p style="text-align: right">
+~780MB</p>
+
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Interactive Diagnosis during Image Reading</strong>
+   </td>
+   <td><p style="text-align: right">
+1 Study or 1 Series</p>
+
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Image QA @ Acquisition</strong>
+   </td>
+   <td><p style="text-align: right">
+1 Series</p>
+
+   </td>
+   <td><p style="text-align: right">
+2048 x 2028</p>
+
+   </td>
+   <td><p style="text-align: right">
+12</p>
+
+   </td>
+   <td>> 30 minutes
+   </td>
+   <td><p style="text-align: right">
+~8MB</p>
+
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Metadata QA</strong>
+   </td>
+   <td><p style="text-align: right">
+1 series or 1 slice</p>
+
+   </td>
+   <td><p style="text-align: right">
+512 x 512</p>
+
+   </td>
+   <td><p style="text-align: right">
+12</p>
+
+   </td>
+   <td>&lt; 500 ms
+   </td>
+   <td><p style="text-align: right">
+~2MB</p>
+
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Retrospective Study</strong>
+   </td>
+   <td><p style="text-align: right">
+100,000 Studies</p>
+
+   </td>
+   <td><p style="text-align: right">
+512x512x600</p>
+
+   </td>
+   <td><p style="text-align: right">
+16</p>
+
+   </td>
+   <td>7 days
+   </td>
+   <td><p style="text-align: right">
+~260MB x 1000</p>
+
+   </td>
+  </tr>
+</table>
+
+
+
+## Computational Workloads {#computational-workloads}
+
+
+<table>
```

Comment:
😥 more HTML in markdown.... 😪

Reply:
See comment above about the Tables and conversion tool problem